streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -327,8 +326,6 @@ public void shouldHandleRemovedTasksToRecycleFromStateUpdater() {
         taskManager = setUpTaskManager(StreamsConfigUtils.ProcessingMode.AT_LEAST_ONCE, true);
         expect(activeTaskCreator.createActiveTaskFromStandby(eq(task01), eq(taskId01Partitions), eq(consumer)))
             .andStubReturn(task01Converted);
-        activeTaskCreator.closeAndRemoveTaskProducerIfNeeded(anyObject());
-        expectLastCall().once();
```

Comment:
Just a heads up: We are currently migrating from EasyMock to Mockito. There is already PR https://github.com/apache/kafka/pull/12607 up to migrate this test. That means the all these mock setups will change in the near future.

Reply:
Thanks. Yes, I guess there will be lots of conflicts anyways, once we are ready to push 4.0 to trunk. So I'll just have to deal with it I guess